tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -423,6 +434,167 @@ RHODS Notification Drawer Should Contain
     Click Element    xpath=//*[contains(@class,'notification-badge')]
     Page Should Contain Element
     ...    xpath=//*[contains(text(),'${message}')]
+    Close Notification Drawer
+
+Open Notebook Images Page
```

Comment:
There are keywords about "Notebook Images", "Image" and "Custom Image". If they are referring to the same I suggest to unify the naming

Reply:
Done